### PR TITLE
Docs: Resource management - Latency/throughput, ACS size

### PR DESCRIPTION
**Pull request by katmurp**
* New doc section for Resource Management in Daml design.
* This PR sets up the top level section page and adds the doc about ACS size. 
* Docs content and input provided by @mziolekda, @miklos-da, and @S11001001.
* Trello ticket here: https://trello.com/c/KJESunOH
* Google doc: https://docs.google.com/document/d/18MyOW2lmCfFdF52oO9g3uC6GA0vAlWiKJkawsxkKMAg/edit#heading=h.okv6bv9jmo1p

### CODE REVIEW

**Review by S11001001 — APPROVED** [review]
Section on JSON API LGTM.

File: docs/source/daml/resource-management/index.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.. Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Resource Management in Daml Application Design
+##############################################
```
Comment (stefanobaghino-da) [inline docs/source/daml/resource-management/index.rst:5]: @remyhaemmerle-da Not something that needs to be addressed right now, but this possibly a good area in which you and your team can contribute with known pitfalls in designing and implementing smart contracts in Daml.

**Comment by katmurp** [comment]
Added the Latency & Throughput section @bame-da

**Comment by katmurp** [comment]
An OK from @mziolekda for the ACS section would be awesome.

**Review by mziolekda — APPROVED** [review]
ACS write-up is OK

**Pull request merged**